Tests/RegressionTests.cs

```diff
@@ -29,7 +29,7 @@ public void BasicTemplateAlgorithm()
                 {"Total Trades", "1"},
                 {"Average Win", "0%"},
                 {"Average Loss", "0%"},
-                {"Compounding Annual Return", "311.484%"},
+                {"Compounding Annual Return", "280.522%"},
```

Comment:
do we know why these were revised lower? just curious. I would have suspected our default statistics mode to match the old method.